lib/generate.js

```diff
@@ -77,6 +77,7 @@ function generateLayoutUnique(options, callback) {
  * @param   {boolean}               [options.maxIconSize]          optional, overrides the max_size in mapnik
  * @param   {boolean}               [options.removeOversizedIcons] optional, if set, filters out icons that mapnik says are too big
  * @param   {boolean}               [options.stretchMetadata]      optional, defaults to true; if set to false, omits stretch metadata (`content`, `stretchX`, `stretchY`) in {@link DataLayout}
+ * @param   {boolean}               [options.concurrency]          optional, defaults to undefined, which means concurrency is unbounded
```

Comment:
```suggestion
 * @param   {number}               [options.concurrency]          optional, defaults to undefined, which means concurrency is unbounded
```

Reply:
The new option should be added to `generateLayout` and `generateLayoutUnique` as well, right?

Reply:
Yes, done in d10b3b5, and went ahead and exposed compression since I'm here.

Reply:
thanks!